tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -118,36 +113,32 @@ private void Dispose(bool disposing)
             }
         }
 
-        private ClusterType MapToplogyToClusterType(BsonValue topology)
+        private void Run(BsonDocument shared, BsonDocument test)
         {
-            switch (topology.AsString)
+            if (shared.TryGetValue("runOn", out var runOn) || shared.TryGetElement("topology", out var topology))
```

Comment:
The test for skipReason used to be first. Is there a reason you changed the order?

Reply:
No, I will move it to the first place.
Done